pop/test.sh

```diff
@@ -411,10 +411,10 @@ EOF
 Location = "Earth, City$n"
 EOF
 		echo "[[parties.servers]]" >> pop_merge.toml
-		sed -n "$((5*$n-4)),$((5*$n))p" public.toml >> pop_merge.toml
+		sed -n "$((5*$n-3)),$((5*$n))p" public.toml >> pop_merge.toml
```

Comment:
These gave me incredible difficulty the last time I had to debug them. How about the next time someone has to touch them, they find a better solution than these fiddly undocumented integres? I promise if it's me I'll do it!

Reply:
I think the integers are OK - I would care more about having a way to correctly cut up the `public.toml` into different servers and then merge them. In fact, we _do_ have that...

Reply:
OK, added it to the code.